src/fighters/Balrog.rb

```diff
@@ -4,6 +4,7 @@
 class Balrog < Fighter 
   def initialize ai
     super ai
+    
```

Comment:
Most action my GitHub has seen in weeks. We're all Bitbucket and JIRA.

I miss GitHub :cry:

Reply:
Cannae decide if I want that line break in or not.

Reply:
Trying to set up Gitlab to run tests automatically. Webhooks aren't doing what I want them to do. Yet.